src/google/google-calendars/googlecalendarsyncadaptor.cpp

```diff
@@ -1241,327 +1305,351 @@ void GoogleCalendarSyncAdaptor::finishedRequestingRemoteEvents(int accountId, co
         return; // still waiting for more requests to finish.
```

Comment:
eventsFinishedHandler() may call this will updateTimestampStr=QString() in the error case and the empty string is still added to m_calendarsFinishedRequested; is that okay?